short_answer/static/css/short_answer.css

```diff
@@ -32,12 +32,20 @@
 }
 
 .short-answer-block .modal {
-    width: 1000px;
+    position: fixed;
+    z-index: 100;
+    display: none;
+    width: 80%;
+    height: 75%;
+    overflow: hidden;
+    left: 10% !important; /* important is necessary to override modal JS styling */
+    margin-left: 0 !important; /* important is necessary to override modal JS styling */
 }
 
 .short-answer-block .modal .inner-wrapper {
-    height: 800px;
+    height: 100%;
     overflow-y: scroll;
+    padding: 0px !important;
 }
```

Comment:
Can you elaborate on how that will remove the need for `!important`? I believe a better fix would be to use a more specific selector https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
https://jsfiddle.net/vmex3hk5/
Like this?

Reply:
If the selector for the rule that this `!important`-ed rule is overriding has a more/same specific selector then it doesn't matter: https://jsfiddle.net/ykegg6wt/
Making the selector more specific is the best solution to avoid `!important` here IMO. (plus adding complexity by generating unique IDs doesn't help)

Reply:
IDs is to avoid using ID in the XBlock. More XBlocks can be present on the same page.
You shouldn't have two elements with the same ID on the same page (Courseware, Studio, etc.)

Reply:
I thought we were talking about _To avoid using !import, we should somehow make a unique id for each xblock_.